server/routes/verifyAddress.ts

```diff
@@ -1,10 +1,10 @@
+import sgMail from '@sendgrid/mail';
```

Comment:
Is all this `verifyAddress` fixes just new linting? Or is there another reason this route has been modified?